lib/rules/indent.js

```diff
@@ -1126,10 +1141,15 @@ module.exports = {
                  */
 
                 const tokenAfterOperator = sourceCode.getTokenAfter(operator);
+                const tokenBeforeOperator = sourceCode.getTokenBefore(operator);
 
-                offsets.ignoreToken(operator);
-                offsets.ignoreToken(tokenAfterOperator);
-                offsets.setDesiredOffset(tokenAfterOperator, operator, 0);
+                if (node.type === "LogicalExpression" && options.LogicalExpression !== DEFAULT_LOGICAL_EXPRESSION_INDENT) {
+                    offsets.setDesiredOffset(operator, tokenBeforeOperator, options.LogicalExpression);
```

Comment:
I think this should be correct:

```js
/* eslint indent: ["error", 4, { LogicalExpression: 1 }] */

const foo = bar
    || baz
    || qux
        .a()
        .b()
    || quux;

```
Or maybe this:

```js
/* eslint indent: ["error", 4, { LogicalExpression: 1 }] */

const foo = bar
    || baz
    || qux
          .a()
          .b()
    || quux;

```




But the rule enforces this:

```js
/* eslint indent: ["error", 4, { LogicalExpression: 1 }] */

const foo = bar
    || baz
    || qux
    .a()
    .b()
        || quux;

```

I'm not too familiar with this rule, but perhaps setting offset relative to the previous token isn't the right solution?

Reply:
Thanks for sharing, I'll try to fix these cases.